source/source_stats_reporter.go

```diff
@@ -35,6 +35,12 @@ var (
 		stats.UnitDimensionless,
 	)
 
+	// RetryEventCountM is a counter which records the number of events sent by the source in retries.
+	RetryEventCountM = stats.Int64(
```

Comment:
just to check: does this need to be exported? (I notice `eventCountM` isn't and I'm wondering if this can be private too)

Reply:
you're right it doesn't have to be!